README.md

```diff
@@ -21,6 +21,9 @@ await normalizeNodeVersion('v8.5.0') // '8.5.0'
 await normalizeNodeVersion('8.5.2') // Error: Invalid Node version
```

Comment:
It might be add to change the text above `Example` with:

```
Normalize and validate Node.js versions.

Can guess the current project's version using its `.nvmrc`.
```